src/cli/cli.cpp

```diff
@@ -8355,6 +8356,57 @@ void Interpreter::HandleIp6Receive(otMessage *aMessage, void *aContext)
 }
 #endif
 
+#if OPENTHREAD_CONFIG_VERHOEFF_CHECKSUM_ENABLE
+
+template <> otError Interpreter::Process<Cmd("verhoeff")>(Arg aArgs[])
+{
+    otError error;
+
+    /**
+     * @cli verhoeff calculate <decimalstring>
```

Comment:
can remove <decimalstring> from the @cli tag, as it doesn't show up in the output anyway. But please include parameters with the @cparam tag.

Reply:
Updates in new push. Thanks.